### PR TITLE
Admin kubernetes content review5 #8361 ke

### PR DESCRIPTION
**Pull request by keichwa**

### CODE REVIEW

File: modules/administration/pages/kubernetes.adoc [hunk 1]
```diff
@@ -47,25 +45,24 @@ _Kubernetes_ clusters are registered with SUSE Manager as ``virtual host manager
 Registration and authorization begins with importing a `kubeconfig` file using Kubernetes official command line tool ``kubectl``.
 
 .Procedure: Registering a Kubernetes Cluster with {productname}
-. Select menu:Systems[Virtual Host Managers] from the navigation menu.
-. Expand the `Create` dropdown in the upper right corner of the page and select menu:Kubernetes Cluster[] .
-. Input a label for the new Virtual Host Manager.
-. Select the `kubeconfig` file which contains the required data for the Kubernetes cluster.
+. Select menu:Systems[Virtual Host Managers] from the main navigation menu of the {webui}.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: I've been using the phrasing "In the {productname} {webui}, navigate to menu:foo[bar].

File: modules/administration/pages/kubernetes.adoc [hunk 2]
```diff
@@ -47,25 +45,24 @@ _Kubernetes_ clusters are registered with SUSE Manager as ``virtual host manager
...
+. Expand the `Create` drop-down menu in the upper right corner and select [guimenu]``Kubernetes Cluster``.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
. In the [guimenu]``Create`` menu, select [guimenu]``Kubernetes Cluster``.
```

File: modules/administration/pages/kubernetes.adoc [hunk 3]
```diff
@@ -47,25 +45,24 @@ _Kubernetes_ clusters are registered with SUSE Manager as ``virtual host manager
...
+. Enter a label for the new virtual host manager.
+. Select the `kubeconfig` file that Contains the required data for the Kubernetes cluster.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
. Select the `kubeconfig` file that contains the required data for the Kubernetes cluster.
```

File: modules/administration/pages/kubernetes.adoc [hunk 4]
```diff
@@ -75,21 +72,20 @@ If the node data is not available wait a few moments and try again.
 
 See the following steps to find runtime data for images.
 
-. Select menu:Images[Images] from the navigation menu.
-. In the image list table, take notice of the new runtime columns.
-These are labeled: ``Revision``, `Runtime` and ``Instances``.
-Initially these columns will not provide useful data.
+. Select menu:Images[Image List] from the navigation menu.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
. In the {productname} {webui}, navigate to menu:Images[Image List].
```

File: modules/administration/pages/kubernetes.adoc [hunk 5]
```diff
@@ -75,21 +72,20 @@ If the node data is not available wait a few moments and try again.
...
+*** _All instances are consistent with SUSE Manager_: All the running instances are running the same build of the image as tracked by {productname}.
+*** _Outdated instances found_: Some of the instances are running an older build of the image.
 A redeploy of the image into the pod may be required.
-*** No information: The checksum of the instance image does not match the image data contained in SUSE Manager.
+*** _No information_: The checksum of the instance image does not match the image data contained in {productname}.
 A redeploy of the image into the pod may be required.
-** ``Instances``: Number of instances running this image across all the clusters registered in SUSE Manager.
-A breakdown of numbers can be seen by clicking on the pop-up icon next to the number.
+** ``Instances``: Number of instances running this image across all the clusters registered in {productname}.
+A breakdown of numbers can be seen by clicking the pop-up icon next to the number.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: I don't understand the nesting here. If you're nesting three deep in a procedure, the procedure needs a re-write.

File: modules/administration/pages/kubernetes.adoc [hunk 6]
```diff
@@ -100,9 +96,9 @@ A breakdown of numbers can be seen by clicking on the pop-up icon next to the nu
 The following steps will help you build an image for deployment in Kubernetes.
 
 . Under menu:Images[Stores], create an image store.
-. Under menu:Images[Profiles], create an image profile (with a Dockerfile which is suitable to deploy to Kubernetes).
+. Under menu:Images[Profiles], create an image profile (with a Dockerfile that is suitable to deploy to Kubernetes).
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
. In menu:Images[Profiles], create an image profile (with a Dockerfile that is suitable to deploy to Kubernetes).
```

File: modules/administration/pages/kubernetes.adoc [hunk 7]
```diff
@@ -116,7 +112,7 @@ The following steps will guide you through importing a previously deployed image
 . Select an image that has already been deployed to any of your registered Kubernetes clusters.
 . Add the registry owning the image to SUSE Manager as an image store.
 . Select menu:Images[Images] , click `Import` from the top-right corner, fill in the form fields and click ``Import``.
-. Notice the updated data in `Runtime` and `Instances` columns in the respective image row.
+. Notice the updated data in the `Runtime` and `Instances` columns in the respective image row.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: This is a result, not a step.

File: modules/administration/pages/kubernetes.adoc [hunk 8]
```diff
@@ -126,25 +122,28 @@ The following steps will guide you through importing a previously deployed image
 
 The following steps will help you find additional runtime data.
 
-. Select to menu:Images[Images] , click the `Details` button on the right end of a row which has running instances.
-. Under the `Overview` tab, notice the data in `Runtime` and `Instances` fields under `Image Info` section.
+. Select menu:Images[Image List].
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: You navigate to a menu item. This and the next step should be together:

. Navigate to menu:{Image List], locate the row that contains the running instance, and click btn:[Details].

(or something)

File: modules/administration/pages/kubernetes.adoc [hunk 9]
```diff
@@ -126,25 +122,28 @@ The following steps will guide you through importing a previously deployed image
...
+. Click the `Details` button on the right end of a row that has running instances.
+. Under the `Overview` tab, notice the data in `Runtime` and `Instances` fields under the `Image Info` section.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: Again, this is a result, not a step.

File: modules/administration/pages/kubernetes.adoc [hunk 10]
```diff
@@ -126,25 +122,28 @@ The following steps will guide you through importing a previously deployed image
...
 . Select the `Runtime` tab.
-. Here is a breakdown of the Kubernetes pods running this image in all the registered clusters including the following data:
+. In the `Runtime` tab is a breakdown of the Kubernetes pods running this image in all the registered clusters including the following data:
++
 
 ** Pod name
 ** Namespace which the pod resides in
 ** The runtime status of the container in the specific pod.
-Status icons are explained in the preceeding example.
+Status icons are explained in the next section.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: For more about status icons, see

(one day I will eliminate all the directional language! Today is not that day ...)

File: modules/administration/pages/kubernetes.adoc [hunk 11]
```diff
@@ -126,25 +122,28 @@ The following steps will guide you through importing a previously deployed image
...
 
 
 
 [[at.rebuild.a.previously.deployed.image.in.kubernetes]]
 == Rebuild a Previously Deployed Image in Kubernetes
 
 
-The following steps will guide you through rebuilding an image which has been deployed to a Kubernetes cluster.
+The following steps will guide you through rebuilding an image that has been deployed to a Kubernetes cluster.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
These steps will guide you through rebuilding an image that has been deployed to a Kubernetes cluster.
```

File: modules/administration/pages/kubernetes.adoc [hunk 12]
```diff
@@ -126,25 +122,28 @@ The following steps will guide you through importing a previously deployed image
...
 
-. Go to menu:Images[Images] , click the Details button on the right end of a row which has running instances.
+. Go to menu:Images[Image List].
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]: These two steps should be together (same as L125)

File: modules/administration/pages/kubernetes.adoc [hunk 13]
```diff
@@ -156,13 +155,13 @@ The image must be manager-built.
 
 [IMPORTANT]
 ====
-Currently, only kubeconfig files containing all embedded certificate data may be used with SUSE Manager
+Currently, only kubeconfig files containing all embedded certificate data may be used with {productname}.
```
Comment (Loquacity) [inline modules/administration/pages/kubernetes.adoc]:
```suggestion
Currently, only kubeconfig files containing all embedded certificate data can be used with {productname}.
```

File: modules/administration/pages/kubernetes.adoc (hunk 5 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: Most of this probably belongs into the Reference guide.

Comment (Loquacity) [reply modules/administration/pages/kubernetes.adoc]: Possibly. I think we should probably create an issue and follow up on this one. It's going to need more work than we can really do in this review.

**Comment by keichwa** [comment]
On 2019-07-26 07:50, Lana Brindley wrote:

> Possibly. I think we should probably create an issue and follow up on
> this one. It's going to need more work than we can really do in this
> review.

For sure.

Nobody dares to touch the Reference these days.

-- 
Karl Eichwalder

**Comment by Loquacity** [comment]
> On 2019-07-26 07:50, Lana Brindley wrote: Possibly. I think we should probably create an issue and follow up on this one. It's going to need more work than we can really do in this review.
> For sure. Nobody dares to touch the Reference these days.

I admit, I'm a little daunted by how much work it needs. I want to make sure we approach that work in an organised way.

**Comment by keichwa** [comment]
On 2019-07-26 08:15, Lana Brindley wrote:

> I admit, I'm a little daunted by how much work it needs. I want to
> make sure we approach that work in an organised way.

It is worth to talk about the Reference.  I'd like to hear what you
both think about it (and, maybe, also the product owner).  In the
past, it was very clear to me how it should look.

-- 
Karl Eichwalder

**Comment by Loquacity** [comment]
> On 2019-07-26 08:15, Lana Brindley wrote: I admit, I'm a little daunted by how much work it needs. I want to make sure we approach that work in an organised way.
> It is worth to talk about the Reference. I'd like to hear what you both think about it (and, maybe, also the product owner). In the past, it was very clear to me how it should look.

Let's discuss at the meeting today.

File: modules/administration/pages/kubernetes.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: thanke!

File: modules/administration/pages/kubernetes.adoc (hunk 5 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: ok, committed a minor simplification.  Ready to be pushed.

File: modules/administration/pages/kubernetes.adoc (hunk 7 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: ok, ready to be pushed

File: modules/administration/pages/kubernetes.adoc (hunk 8 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: ok, ready to be merged

File: modules/administration/pages/kubernetes.adoc (hunk 10 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: ok, thanks

File: modules/administration/pages/kubernetes.adoc (hunk 12 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: ok

File: modules/administration/pages/kubernetes.adoc (hunk 9 above)
Comment (keichwa) [reply modules/administration/pages/kubernetes.adoc]: ok, combining with the preceding step.

**Comment by keichwa** [comment]
I hope everything is in now. Ready for review!

**Pull request merged**